<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DBSCAN Visualizer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Density Based Scan Algorithm</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,1,0">
       <item row="2" column="1">
        <widget class="SpinBox" name="minDensitySpin"/>
       </item>
       <item row="3" column="0" colspan="4">
        <widget class="TableWidget" name="clusterTable">
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="columnCount">
          <number>6</number>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>60</number>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>20</number>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>20</number>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>10</number>
         </attribute>
         <column>
          <property name="text">
           <string>N Points</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Centroid</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Grid Area</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Box Area</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Density</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Avg Distance</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="exportButton">
         <property name="text">
          <string>Export All</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="SpinBox" name="minNeighborsSpin"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Minimum points per cluster</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="SpinBox" name="epsilonSpin"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Epsilon Distance</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Minimum Neighbors</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="clusterButton">
         <property name="text">
          <string>Cluster</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Table shows up to 100 items</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="DataWidget" name="tableWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>383</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>Export</string>
    </property>
    <addaction name="actionExportDistances"/>
    <addaction name="actionExportClusters"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuExport"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionNo_Recent_Files">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>No Recent Files</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionExportDistances">
   <property name="text">
    <string>Cluster Distances</string>
   </property>
  </action>
  <action name="actionExportClusters">
   <property name="text">
    <string>Cluster Data</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>pyqtgraph</header>
  </customwidget>
  <customwidget>
   <class>TableWidget</class>
   <extends>QTableWidget</extends>
   <header>pyqtgraph</header>
  </customwidget>
  <customwidget>
   <class>DataWidget</class>
   <extends>QTableWidget</extends>
   <header>DataWidget</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
